src/js/components/SavedSpacesList.js

```diff
@@ -37,33 +38,40 @@ export default function SavedSpacesList({spaces, spaceContextMenu}: Props) {
 
   return (
     <menu className="saved-spaces-list">
-      {spaces.map(brim.space).map((s) => {
-        const progress = s.ingesting() && (
-          <div className="small-progress-bar">
-            <ProgressIndicator percent={s.ingestProgress()} />
-          </div>
-        )
+      {spaces
+        .sort((a, b) => (a.name > b.name ? 1 : -1))
+        .map(brim.space)
+        .map((s) => {
+          const progress = s.ingesting() && (
+            <div className="small-progress-bar">
+              <ProgressIndicator percent={s.ingestProgress()} />
+            </div>
+          )
+          return (
+            <li key={s.id}>
+              <a
+                href="#"
+                onClick={onClick(s.id)}
+                onContextMenu={() => {
+                  !s.ingesting() &&
+                    showContextMenu(spaceContextMenu(s.id, s.name))
+                }}
+                className={classNames("space-link", {
+                  "current-space-link": s.id === currentSpaceId
+                })}
+              >
+                {s.storage_kind === "archivestore" ? (
```

Comment:
👍

Reply:
I'm predicting we will want to refactor this a little once we add dropdown functionality and hierarchy to the spaces